jobs/openshift/cluster/integreatly/test/Jenkinsfile

```diff
@@ -0,0 +1,109 @@
+#!groovy
+
+def testOptions = [:]
+
+def verifyParameters(testOptions) {
+  if (!testOptions.gitUrl || !testOptions.gitBranch) {
+    def userInput = input message: 'Integreatly Test Options', parameters: [
+      string(defaultValue: (testOptions.gitUrl ?: 'https://github.com/integr8ly/installation.git'), description: 'Integreatly installer Git URL', name: 'installerGitUrl'),
+      string(defaultValue: (testOptions.gitBranch ?: ''), description: 'Integreatly Git Branch', name: 'installerGitBranch'),
+    ]
+
+    testOptions.gitUrl = userInput.installerGitUrl
+    testOptions.gitBranch = userInput.installerGitBranch
+    testOptions.clusterName = userInput.installerGitBranch
+    testOptions.openshiftMasterUrl = "https://${userInput.installerGitBranch}.${testOptions.domainName}"
+    verifyParameters(testOptions)
+  }
+}
+
+def buildJob(jobName, jobParams) {
+  build job: jobName, parameters: jobParams
+}
+
+stage('Verify Parameters') {
+  testOptions.adminUsername = 'integreatly'
+  testOptions.adminPassword = 'Password1'
+  testOptions.awsRegion = 'eu-west-2'
+  testOptions.awsAccountName = 'fheng.AWS'
+  testOptions.clusterName = params.installerGitBranch
+  testOptions.domainName = 'skunkhenry.com'
+  testOptions.gitUrl = params.installerGitUrl
+  testOptions.gitBranch = params.installerGitBranch
+  testOptions.openshiftMasterUrl = "https://${params.installerGitBranch}.${testOptions.domainName}"
+  verifyParameters(testOptions)
+}
+
+node('cirhos_rhel7') {
+  cleanWs()
+  stage('Create Cluster') {
+    println "[INFO] Creating ${testOptions.clusterName} cluster"
+    def jobName = 'openshift-cluster-create'
+    def instanceType = 'm4.xlarge'
+    def instanceGroupSize = '3'
```

Comment:
Probably wan't to adjust these once we have the ability to reduce the size of the clusters. Issue on the backlog to look into why that's not working properly at the moment.

Reply:
Probably won;t all be the same either, master should be 1 and infra and compute 2 or more. Should we consider just setting the default values in the create job instead of passing them in here?

Reply:
The current default value of these is set to 3 in our repo but set to the values you mentioned above on Jenkins. Should these be changed to: master(1), infra/compute(2)? I can include that change in this PR. 

If the configuration above would be enough, it would be great to use the default values.

Reply:
I'm thinking we should make the default values in the create job be the minimum that we will require the majority of the time. So probably master(1), infra/compute(2) as you say, so if you just remove all those params from here, when we do eventually update that job (Once we figure out https://issues.jboss.org/browse/INTLY-1675), this job will just do the correct thing.